packages/strapi-plugin-i18n/admin/src/components/CMEditViewCopyLocale/index.js

```diff
@@ -17,13 +17,23 @@ import {
 import { getTrad } from '../../utils';
 import { cleanData, generateOptions } from './utils';
 
-const CMEditViewCopyLocale = ({ appLocales, currentLocale, localizations, readPermissions }) => {
+const CMEditViewCopyLocale = props => {
+  if (!props.localizations.length) {
+    return null;
+  }
+
+  return <Content {...props} />;
+};
+
+const Content = ({ appLocales, currentLocale, localizations, readPermissions }) => {
+  const options = generateOptions(appLocales, currentLocale, localizations, readPermissions);
+
   const { formatMessage } = useIntl();
   const dispatch = useDispatch();
   const { allLayoutData, slug } = useContentManagerEditViewDataManager();
   const [isLoading, setIsLoading] = useState(false);
   const [isOpen, setIsOpen] = useState(false);
-  const [value, setValue] = useState(null);
+  const [value, setValue] = useState(options[0]);
```

Comment:
Do `options[0]` always defined?

Reply:
Yes this is why I have split the component into two parts.